community/kernel/src/test/java/org/neo4j/graphdb/NativeLabelScanStoreUpdateIT.java

```diff
@@ -209,7 +214,10 @@ public void shouldHandleLargeAmountsOfNodesAddedAndRemovedInSameTx() throws Exce
         }
     }
 
-    protected abstract GraphDatabaseService db();
+    protected GraphDatabaseService db()
```

Comment:
we do not need this method to be protected anymore, mostly we do not need it at all